src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import './blocks/header';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  font-family: $font-family;
+  font-size: 12px;
+  font-weight: 500;
 }
```

Comment:
Consider moving these styles into a separate file with general styles to keep your main file clean and clear